source/data/polls.js

```diff
@@ -1,107 +1,108 @@
-const { db } = require('../../dbConnection.js');
-const logger = require('../../logFunctions');
-db.connect()
-    .catch(error => {
-        // Db Connect Throws errors even when it works, 
-        // It tries to authenticate with computer credentials before trying the .env credentials
-        // console.log(error);
+const { db } = require('../../dbConnection.js')
+const logger = require('../../logFunctions')
+db.connect().catch(error => {
+  // Db Connect Throws errors even when it works,
+  // It tries to authenticate with computer credentials before trying the .env credentials
+  // console.log(error);
 })
 
 const getAll = async () => {
-    db.query('SELECT * FROM polls where is_closed = false ORDER BY id ASC')
-        .then((results) => {
-            return results.rows;
-        });
-};
+  db.query('SELECT * FROM polls where is_closed = false ORDER BY id ASC').then(results => {
+    return results.rows
+  })
+}
 
-const getByChannel = async (req) => {
-    const {
-        teamId,
-        channelId,
-        isClosed
-    } = req;
-    return db.query('SELECT * FROM polls where team_id= $1 and channel_id =$2 and is_closed = $3 LIMIT 1',
-        [teamId, channelId, isClosed]).then((results) => {
-        return results.rows[0];
-    });
-};
+const getByChannel = async req => {
+  const { teamId, channelId, isClosed } = req
+  return db
+    .query('SELECT * FROM polls where team_id= $1 and channel_id =$2 and is_closed = $3 LIMIT 1', [
+      teamId,
+      channelId,
+      isClosed
+    ])
+    .then(results => {
+      return results.rows[0]
+    })
+}
 
-const getByChannelWithResults = async (req) => {
-    const {
-        teamId,
-        channelId,
-        isClosed
-    } = req;
-    return db.query('SELECT *, poll_votes.id as poll_vote_id, poll_options.id as poll_options_id FROM polls INNER JOIN poll_options ON polls.id = poll_options.poll_id LEFT JOIN poll_votes ON poll_options.id = poll_votes.option_id LEFT JOIN users ON poll_votes.voter_id = users.id where polls.team_id= $1 and polls.channel_id =$2 and polls.is_closed = $3',
-    [teamId, channelId, isClosed]).then((results) => {
-        return results.rows;
-    });
-};
+const getByChannelWithResults = async req => {
+  const { teamId, channelId, isClosed } = req
+  return db
+    .query(
+      'SELECT *, poll_votes.id as poll_vote_id, poll_options.id as poll_options_id FROM polls INNER JOIN poll_options ON polls.id = poll_options.poll_id LEFT JOIN poll_votes ON poll_options.id = poll_votes.option_id LEFT JOIN users ON poll_votes.voter_id = users.id where polls.team_id= $1 and polls.channel_id =$2 and polls.is_closed = $3',
```

Comment:
This line definitely exceeds 100 characters, If you're fixing formatting please break up this line.